packages/location-management/src/ducks/location-units.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  fetchActionCreatorFactory,
+  removeActionCreatorFactory,
+  setTotalRecordsFactory,
+  reducerFactory,
+  getItemsByIdFactory,
+  getItemsArrayFactory,
+  getItemByIdFactory,
+  getTotalRecordsFactory,
+} from '@opensrp/reducer-factory';
+
+/** Enum representing the possible location unit status types */
+export enum LocationUnitStatus {
+  ACTIVE = 'Active',
+  INACTIVE = 'InActive',
+}
+
+/** interface for LocationUnit.properties */
+export interface Properties {
+  geographicLevel: number;
+  name: string;
+  parentId: string;
+  status: LocationUnitStatus;
+  username: string;
+  version: number;
+  name_en: string;
+  externalId: string;
+  OpenMRS_Id: string;
+}
+
+/** location tag interface */
+export interface LocationTag {
+  id: string | number;
+  name: string;
+}
+
+/** geometry interface */
+export interface Geometry {
+  type: string;
+  coordinates: Array<Array<Array<Array<number>>>>;
```

Comment:
Does this cover the whole range of coordinates that we might handle, for example, would coordinates of a single point be correctly typed by the type `Array<Array<Array<Array<number>>>>`?

Reply:
@dubdabasoduba I forgot to confirm the format of the `coordinates` as presented in this [sample payload](https://github.com/OpenSRP/web/issues/53) cc @p-netm

Reply:
@dubdabasoduba Is `coordinates` a nested array?

Reply:
Yes it is https://github.com/OpenSRP/web/issues/53 has the sample payloads

Reply:
Yes, it can be a nested array or just a point that has `longitude` & `latitude` or blank for instances that don't pick GPS coordinates

Reply:
@kelvin-muchiri I think this might help https://github.com/DefinitelyTyped/DefinitelyTyped/blob/22043e9b09764c10ff16aac9bb53d6fd2bd1e2ab/types/geojson/index.d.ts#L79

Reply:
@p-netm I have changed to use `Geometry` from `@turf/turf`